integration

```diff
@@ -9,6 +9,7 @@
   "aceindy/Duepi_EVO",
   "acesyde/hassio_mylight_integration",
   "adamoutler/anycubic-homeassistant",
+  "adechant/ha_vscode",
```

Comment:
The documentation only describes how to download/configure.
There is no descriptions of what this actually does (like downloading and running a binary and exposing your entire system to vscode, as this is unexpected behavior of integrations (generally only add-ons do this), I would expect to see information/warnings about it).